### PR TITLE
[docs] add config tag

### PR DESCRIPTION
**Pull request by congee524**

### CODE REVIEW

File: configs/detection/ava/README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
 
 ## Introduction
 
-```
+[ALGORITHM]
+
+```citation
```
Comment (innerlee) [inline configs/detection/ava/README.md:11]: is `citation` valid?

**Comment by innerlee** [comment]
backbones are missing

File: configs/detection/ava/README.md (hunk 1 above)
Comment (congee524) [reply configs/detection/ava/README.md]: have been fixed

**Comment by innerlee** [comment]
Pls also update the stats.py

**Comment by congee524** [comment]
> backbones are missing

we don't have backbones. The current statistics are based on each folder.

C3D and I3D is only applied in video understanding, they are considered as algorithms.

**Comment by innerlee** [comment]
What happens to papers with ` " " `

image

**Comment by congee524** [comment]
> What happens to papers with `" "`
> 
> image

Since each model can only be counted for one paper, the operators and datasets involved in the model implementation should not be counted, so double quotation marks are added to ignore it.

**Comment by congee524** [comment]
> Pls also update the stats.py

image

**Comment by congee524** [comment]
image

**Review by innerlee — APPROVED** [review]

**Pull request merged**